runtime/compiler/ilgen/J9ByteCodeIlGenerator.hpp

```diff
@@ -197,7 +197,7 @@ class TR_J9ByteCodeIlGenerator : public TR_IlGenerator, public TR_J9ByteCodeIter
    TR::SymbolReference *loadClassObjectForTypeTest(int32_t cpIndex, TR_CompilationOptions aotInhibit);
    void         loadClassObject(TR_OpaqueClassBlock *opaqueClass);
    void         loadArrayElement(TR::DataType dt){ loadArrayElement(dt, comp()->il.opCodeForIndirectArrayLoad(dt)); }
-   void         loadArrayElement(TR::DataType dt, TR::ILOpCodes opCode, bool checks = true);
+   void         loadArrayElement(TR::DataType dt, TR::ILOpCodes opCode, bool checks = true, bool isNotValueType = false);
```

Comment:
I find the default negative naming to be a bit confusing.  How about naming this `bool mayBeValueType = true` ?  The presumption being that it may be a value type until proven otherwise.  If you agree, the consumers of this parameter will have to flip their polarity.

Reply:
Updated in fa98a9c